codedeploy/be/scripts/validate.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+# scripts/validate.sh
+
+for i in {1..10}; do
+  HTTP_CODE=$(curl -s -o /dev/null -w "%{http_code}" <http://localhost:8080/actuator/health>)
```

Shellcheck (0.10.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)